config/clients/go/template/client/errors.mustache

```diff
@@ -19,3 +19,22 @@ func (e FgaRequiredParamError) Error() string {
 func (e FgaRequiredParamError) Param() string {
 	return e.param
 }
+
+// FgaInvalidUlidError Provides access to the body, error and model on returned errors.
+type FgaInvalidUlidError struct {
```

Comment:
Maybe we can make this more generic? e.g. `FgaInvalidFormatError` it takes error, param and format?

Just so that we can reuse it elsewhere (e.g. when we validate urls)

Reply:
done